examples/simple/page.pug

```diff
@@ -0,0 +1,18 @@
+extends ../index.pug
+
+block content
+  include:markdown-it README.md
+  button.btn.btn-primary.btn-sm#page page
+
+block footer_content
+  script.
+    !function(e,t,s,n,a,r,c,i){e[a]=e[a]||function(){(e[a].q=e[a].q||[]).push(arguments)},c=t.createElement(s),i=t.getElementsByTagName(s)[0],
+    c.async=1,c.src=n,c.charset=r,c.setAttribute("data-ud-namespace",a),i.parentNode.insertBefore(c,i)}
+    (window,document,"script","https://cdn.userdive.com/agent.js","_ud","UTF-8");
+    _ud("create", 'af57h6gb', 'auto');
+    _ud('send', 'pageview');
+    document.getElementById('page')
+    .addEventListener('click', function () {
+      _ud('send', 'pageview', { page: location.origin+location.pathname+'/dummy'+location.search });
```

Comment:
v0.15 does not work yet;
https://github.com/userdive/agent.js/blob/v0.15.0/packages/agent/src/agent.ts#L46

Reply:
why not use `_ud('send', 'pageview', pathname)` ?

Reply:
based api syntax https://developers.google.com/analytics/devguides/collection/analyticsjs/pages

Reply:
Yes, i know that agent has this interface and `pathname2href` in core.ts work for too.
But this example describe the `page` field can override like [this](https://developers.google.com/analytics/devguides/collection/analyticsjs/pages#modifying_page_urls).